lib/msal-common/src/client/SPAClient.ts

```diff
@@ -213,7 +213,13 @@ export class SPAClient extends BaseClient {
             // Get current cached tokens
```

Comment:
Some unecessary console.log statements in this file.

Reply:
> We should consider adding tests in both browser and node to catch these types of regressions.

Planning on reviewing this tomorrow morning, need to improve unit testing in common.